### DISCUSSION TITLE
Chapter 5 : Brownie Simple Storage - I'm getting ImportError: cannot import name 'simpleStorage' from 'brownie' error

### DISCUSSION THREAD

**Discussion by ghost** [smartcontractkit/full-blockchain-solidity-course-py#1866] [Q&A]
<img width="695" alt="Screenshot 2023-04-12 at 8 03 04 AM" src="https://user-images.githubusercontent.com/89763258/231333268-f8f9a546-df99-42d7-bc27-a336f61244f0.png">

while doing brownie deploy run deploy.py I get the below error

<img width="388" alt="Screenshot 2023-04-12 at 8 03 19 AM" src="https://user-images.githubusercontent.com/89763258/231333373-6601e6bc-ea36-4ae9-93de-26faaab4dfae.png">

Please help as its saying ImportError: cannot import name 'simpleStorage' from 'brownie' error 

Below is the structure of my file 
https://github.com/raf-1404/filepath2

**Comment by cromewar** [comment]
Hello @raf-1404 
It seems you are importing `simpleStorage` with a lower case S, and the contracts might be `SimpleStorage` with upper case, give it a try.

**Comment by 0xFLT** [comment]
Yes you need to put the name of the compiled contract which is in Browniefolder>Build>Contracts>FileName.json instead of putting the name of the .sol file.
